cms/templates/js/unit-access-editor.underscore

```diff
@@ -0,0 +1,133 @@
+            return;
+        }
+    }
+    selectablePartitions.push(partition);
+});
+
+selectablePartitions.forEach(function (partition) {
+    var schemeId = partition['id'];
+    var groups = partition['groups'];
+    groups.forEach(function (group) {
+        var groupId = group['id'];
+        var groupName = group['name'];
+        var fullId = schemeId.toString() + '-' + groupId.toString();
+        groupIdToName[fullId] = groupName;
+    })
+});
+
+var deletedGroups = [];
+
+for (var schemeId in groupAccess) {
+    groupAccess[schemeId].forEach(function (groupId) {
+        var fullId = schemeId.toString() + '-' + groupId.toString();
+        if (!groupIdToName[fullId]) {
+            deletedGroups.push(fullId);
+        }
+    })
+}
+
+// Need to include deleted groups logic to not change is_active boolean in contentstore/utils.py get_user_partition_info
+deletedGroups.forEach(function (fullId) {
+    var schemeId = fullId.split('-')[0];
+    var groupId = fullId.split('-')[1];
+    var partitionExists = false;
+    selectablePartitions.forEach(function (partition) {
+        if (partition['id'] === schemeId) {
+            partitionExists = true;
+            partition['groups'].push(
+                {
+                    "deleted": true,
+                    "selected": true,
+                    "id": groupId,
+                    "name": ""
+                }
+            );
+        }
+    })
+
+    if (!partitionExists) {
+        selectablePartitions.push(
+            {
+                "scheme": "cohort",
+                "id": schemeId,
+                "groups": [
+                    {
+                        "deleted": true,
+                        "selected": true,
+                        "id": groupId,
+                        "name": "Deleted Group"
+                    }
+                ],
+                "name": "Content Group Configuration"
+            }
+        )
+    }
+})
+%>
+<form>
+<h3 class="modal-section-title">
+    <%- gettext('Unit Access') %>
+</h3>
+<div class="modal-section-content access-change">
+    <label for="partition-select" class="group-select-title"><%- gettext('Restrict access to:') %>
+        <select class="user-partition-select" id="partition-select">
+            <option value="-1" selected ="selected"><%- gettext('Select a group type') %></option>
+            <% for (var i=0; i < selectablePartitions.length; i++) { %>
+                <option id="<%- selectablePartitions[i].id %>-option" value="<%- selectablePartitions[i].id %>"><%- selectablePartitions[i].name %></option>
+            <% } %>
+        </select>
+    </label>
+    <br>
+    <% if (selectablePartitions.length > 0) { %>
+        <div class="user-partition-group-checkboxes">
+            <% for (var i=0; i < selectablePartitions.length; i++) { %>
+                <div id="<%- selectablePartitions[i].id %>-checkboxes">
+                    <div class="partition-group-directions">
```

Comment:
You need to use role="group" and aria-labelledby to link the directions. Make the HTML the same as in visibility_editor.html, as we took care there to handle accessibility concerns.